seedqreader.py

```diff
@@ -592,7 +605,14 @@ def upd_camera_stream(self, frame):
         self.ui.video_in.setPixmap(frame)
 
     def on_slider_move(self):
-        self.ui.split_size.setText(f"Split size: {self.ui.send_slider.value()}")
+        self.ui.split_size.setText(f"QR split size: {self.ui.send_slider.value()}")
+
+    def on_delay_slider_move(self):
+        self.ui.delay_size.setText(f"QR delay: {self.ui.delay_slider.value()}")
+        try:
+            self.display_qr.set_delay(self.ui.delay_slider.value())
+        except:
+            pass
```

Comment:
i don't think we can get an exception if self.display_qr != None?
```suggestion

        if self.display_qr:
            self.display_qr.delay = (self.ui.delay_slider.value())
```